services/ui-src/src/measures/2023/shared/CommonQuestions/OptionalMeasureStrat/ndrSets.tsx

```diff
@@ -100,13 +104,13 @@ const TotalNDRSets = ({
   const { qualifiers, categories } = usePerformanceMeasureContext();
   const totalQual = qualifiers.slice(-1)[0];
 
-  if (categories.length) {
+  if (categories.length && categories?.filter((item) => item.label).length) {
```

Comment:
NIT: I think `.some` would be a little clearer here
```suggestion
  if (categories.length && categories.some((item) => item.label)) {
```